devenv.nix

```diff
@@ -74,6 +74,11 @@
       enable = true;
       settings.preset = "relaxed";
     };
+    pdm_export = {
+        enable = true;
+        name = "PDM Export";
+        entry = "pdm export -o requirements.txt --without-hashes";
```

Comment:
isn't `--without-hashes` agains all benefits of `pdm` or `nix` in general?

Reply:
Will remove the whole requirements.txt